compute/src/test/java/org/jclouds/compute/internal/BaseComputeServiceLiveTest.java

```diff
@@ -386,11 +386,11 @@ public void testCreateTwoNodesWithOneSpecifiedName() throws Exception {
       this.nodes = newTreeSet(concat(this.nodes, nodes));
    }
 
-   private Template refreshTemplate() {
+   protected Template refreshTemplate() {
```

Comment:
@nacx: Similarly to the comment below, if the "standard" code this test is executing doesn't work, does it mean users will also need to execute "non-standard" commands to interact successfully with the image?